client/components/web-preview/index.jsx

```diff
@@ -1,3 +1,5 @@
+/** @ssr-ready **/
+
```

Comment:
Kind of weird to mark this file as `@ssr-ready` since it contains references to `window` and `document`. OTOH, they're wrapped in lifecycle methods, and this PR demonstrate that SSR works, so I guess it's fine :-)